libcxx/include/__type_traits/has_unique_object_representation.h

```diff
@@ -24,7 +22,7 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 template <class _Tp>
 struct _LIBCPP_TEMPLATE_VIS has_unique_object_representations
-    : public integral_constant<bool, __has_unique_object_representations(remove_cv_t<remove_all_extents_t<_Tp>>)> {};
```

Comment:
I assume you've checked that we have tests for const arrays or w/e to make sure the patch is still exercised.

Reply:
Yes to both questions, but I wouldn't be upset if someone wanted to double check the tests' existence.

Reply:
There is `libcxx/test/std/utilities/meta/meta.unary/meta.unary.prop/has_unique_object_representations.pass.cpp`, but we seem to be missing a test for a multi-dimensional array.